### DISCUSSION TITLE
Request failed with status code 400 - For the new user telegram notifications

### DISCUSSION THREAD

**Discussion by bgktmxg** [sct/overseerr#1175] [Help]
Hi guys, can someone maybe tell me where i'm going wrong. When entering chat ID into the user profile telegram notification settings, the user does not get a notification and error 400 in log.

Im on latest v1.21.0. Have tried new chats, existing chats. Even using the chat ID for the currently operational admin notifications.

![image](https://user-images.githubusercontent.com/51685078/111101792-5a35dc80-8596-11eb-901f-396c64455e15.png)

**Comment by sct** [comment]
Someone was having a very similar problem in Discord a few days ago: https://discord.com/channels/783137440809746482/789002037529804833/819712506414366731

Maybe take a look and see if this discussion helps you at all? It seemed to be related to the bot giving them the wrong ID.

**Comment by bgktmxg** [comment]
Ok so im an idiot, but maybe this can serve as a lesson for the other less savvy people here. The bot that you created for the admin notifications must be added to each individual users chat in order for the notifications to work. Im sure this is very obvious for most people but not for me. Maybe there could be notes added to overseer to clarify this?
Cheers

**Reply by sct** [reply]
We can add details about the notification systems when we flesh out our documentation a bit more. Will make a note of this in there!

**Reply by dtalens** [reply]
For 10 minutes I looked like an idiot until I saw your comment.
